### ISSUE TITLE
Loss function definition

### ISSUE THREAD

**Issue by chinmay5** [noaRricky/pytorch-glove#1]
I think that the loss function definition is incorrect since we need to subtract the log of co-occurance counts. Can you please check it once

**Comment by ismaeel123** [comment]
Yes,  i think we should replace the +log with -log

**Comment by giturra** [comment]
Hi guys!

I have a question about the implementation. Here is in the fit method:
```python

word_counts = Counter()
 cooccurence_counts = defaultdict(float)
 for region in corpus:
     word_counts.update(region)
     for left_context, word, right_context in _context_windows(region, left_size, right_size):
         for i, context_word in enumerate(left_context[::-1]):
                # add (1 / distance from focal word) for this pair
                cooccurence_counts[(word, context_word)] += 1 / (i + 1)
         for i, context_word in enumerate(right_context):
                cooccurence_counts[(word, context_word)] += 1 / (i + 1)
```

The ```cooccurrence_counts```  are weighted by the distance from the focal word, but the paper mention that the co-occurrence matrix is filled with the count by the pair target and context word. Is the implementation it okay?
